compiler/src/iree/compiler/ConstEval/Runtime.cpp

```diff
@@ -472,20 +512,28 @@ InMemoryCompiledBinary::translateFromModule(mlir::ModuleOp moduleOp) {
     return failure();
   }
   os.flush();
-  initialize(&binary[0], binary.length());
-  return success();
+  return initialize(moduleOp.getLoc(), &binary[0], binary.length());
 }
 
 Runtime::Runtime() {
-  IREE_CHECK_OK(
-      iree_hal_driver_registry_allocate(iree_allocator_system(), &registry));
-  IREE_CHECK_OK(iree_hal_local_task_driver_module_register(registry));
-  IREE_CHECK_OK(iree_vm_instance_create(IREE_VM_TYPE_CAPACITY_DEFAULT,
-                                        iree_allocator_system(), &instance));
-  IREE_CHECK_OK(iree_hal_module_register_all_types(instance.get()));
+  if (iree_status_is_ok(initStatus)) {
```

Comment:
because you don't default initialize initStatus this is wrong - you should initialize it